Makefile

```diff
@@ -13,26 +11,26 @@ down: ## Bring down docker containers
 	cd fdbt-dev && $(MAKE) docker-down
 
```

Comment:
I have noticed that when I make a change locally it is not reflecting e.g. adding a log to the api it shows the older log I put despite refreshing

Reply:
Resolved